pack/thor.json

```diff
@@ -139,7 +140,7 @@
 		"resource_mental": 1,
 		"set_code": "thor",
 		"set_position": 9,
-		"text": "<b>Hero Action</b>: Spend X [energy] resources → deal X damages to the villain and each minion engaged with you. This attack ignores tough statuses card if you have the [[Aerial]] trait.",
+		"text": "<b>Hero Action</b>: Spend X [energy] resources → deal X damages to the villain and each minion engaged with you. This damage ignores tough statuses card if you have the [[Aerial]] trait.",
```

Comment:
Existing issues but
1. This should say `deal X damage` instead of `deal X damages`.
2. This should say `ignores tough status cards`